src/openrct2/world/scenery.h

```diff
@@ -236,6 +236,14 @@ enum {
     SCENERY_TYPE_BANNER
 };
 
+enum {
+    SCENERY_GHOST_FLAG_0 = (1 << 0),
```

Comment:
Small Scenery, Path, Wall, Large Scenery, Banner

Reply:
So what should I do?

Reply:
Hmm I'd be tempted to remove ghost flags enum and use (1<<SCENERY_TYPE_xxx) instead. The other option would be to name the ghost flags as above. What do you feel is more descriptive and makes the coffee easier to read?

Reply:
I think it would indeed but I was trying to follow [Gymnasiast's advice](https://github.com/OpenRCT2/OpenRCT2/issues/5173#issuecomment-333527437)

Reply:
You can follow Duncan's advice here.